### DISCUSSION TITLE
kafka kraft support in production

### DISCUSSION THREAD

**Discussion by Firas-Zarai** [strimzi/strimzi-kafka-operator#9581] [Q&A] [answered]
hello, 

I want to know if the latest version of streamzi 0.39.0 supports Kafka kraft in production because in the documentation it's mentioned that it didn't support and it has limitations.
https://strimzi.io/docs/operators/latest/deploying#ref-operator-use-kraft-feature-gate-str
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/59801055/c6d31b8c-deb2-45e6-b9ef-962ff72a9082)

thanks in advance.

**Comment by scholzj** [comment, marked as answer]
Duplicate of a Slack discussion: https://cloud-native.slack.com/archives/CMH3Q3SNP/p1706008354391169?thread_ts=1706008354.391169&cid=CMH3Q3SNP. You should ask only in one place at a time!

**Comment by patpatpat123** [comment]
Is kafka kraft still not ready for production?
